InfiniteScrollView.js

```diff
@@ -96,8 +95,11 @@ export default class InfiniteScrollView extends React.Component {
   }
 
   _shouldLoadMore(event) {
+    
+    var canLoadMore = (typeof this.props.canLoadMore == 'function') ? this.props.canLoadMore() : this.props.canLoadMore;
```

Comment:
- let instead of var
- === instead of ==
- can you split this across multiple lines:
```js
let canLoadMore = (typeof this.props.canLoadMore === 'function') ?
  this.props.canLoadMore() :
  this.props.canLoadMore;
```